app/models/user.rb

```diff
@@ -82,6 +82,7 @@ def propagate_user_email_errors
     errors[:'emails.email'].each do |error|
       errors.add(:email, error)
     end
+    errors.delete(:'emails.email')
```

Comment:
@lowjoel  I think this error should be deleted after propagate, otherwise it shows twice.